packages/react/src/stories/ic-side-navigation.stories.mdx

```diff
@@ -510,10 +510,12 @@ export const DailyTippers = () => (
   >
     <div style={{"display": "flex", "height": "100%"}}>
       <IcSideNavigation
-        appTitle="ACME"
         version="v0.0.0"
         status="BETA"
       >
+        <NavLink to="/" slot="app-title">
+          ACME
+        </NavLink>
         <svg
```

Comment:
By only wrapping it around the app title, doesn't that mean it is not possible to click on the app icon when side navigation is closed and be taken to the link destination? 
To be honest, I'm not sure if that was possible in the first place but can imagine users clicking on the app icon to go to the home screen..
Would it be possible to add that functionality into this ticket?